spec/factories/tutor.rb

```diff
@@ -4,4 +4,13 @@
     last_name { Faker::Name.last_name }
     email { Faker::Internet.email }
   end
+
+  factory :full_tutor, class: Tutor do
```

Comment:
The naming scheme I used for students was that a full student had a tutor and a coordinator, so in this case a full tutor would have a student and a coordinator. It might be easier if we stick to the scheme. Something like a tutor or an employed tutor or something would have a coordinator but not student.